src/einsteinpy/plotting/plotlyplotter.py

```diff
@@ -0,0 +1,71 @@
+import os
+import random
+
+import numpy as np
+import plotly.graph_objs as go
+from plotly.offline import plot as saveplot
+
+
+class PlotlyPlotter:
+    def __init__(self, attractor_color="#ffcc00"):
+        """
+        Constructor.
+
+        Parameters
+        ----------
+        attractor_color : string, optional
+            Color which is used to denote the attractor. Defaults to #ffcc00.
+
+        """
+        self.fig = go.FigureWidget()
+        self.attractor_color = attractor_color
+        self.attractor_present = False
+
+    def _draw_attractor(self, radius, xarr, yarr):
+        self.attractor_present = True
+        self.fig.add_trace(
+            go.Scatter(
+                x=[0],
+                y=[0],
+                mode="markers",
+                name="attractor",
+                marker=dict(size=10, color=self.attractor_color, line=dict(width=2)),
+            )
+        )
+
+    def plot(self, geodesic, color="#{:06x}".format(random.randint(0, 0xFFFFFF))):
+        """
+
+        Parameters
+        ----------
+        geodesic : ~einsteinpy.geodesic.Geodesic
+            Geodesic of the body
+        color : hex code RGB, optional
+            Color of the dashed lines. Picks a random color by default.
+
+        """
+        vals = geodesic.trajectory
+        r = np.array([coord[1] for coord in vals])
+        phi = np.array([coord[3] for coord in vals])
+
+        x = r * np.cos(phi)
```

Comment:
This part is wrong! As discussed in #277 
@ritzvik Can you quickly tell how to correct it?

Reply:
I would say that we should use `return_cartesian=True` when calling `calculate_trajectory`

OR

Use spherical to cartesian conversion using the `coordinates` module for `Schwarzschild`
And BL to cartesian for `Kerr`. To see the implementation just look at any one of the `calculate_trajectory` functions!